.github/CODEOWNERS

```
@@ -1,3 +1,3 @@
 * @delapuente @y4izus @techtolentino @korgan00 @eddybrando
-content/advocates/*   @lerongil @aasfaw
-content/events/*      @justjosie @KallieFerguson @delapuente @lerongil @aasfaw
+content/advocates/*   @lerongil
+content/events/*      @justjosie @KallieFerguson @delapuente @lerongil
```

We might also remove `@delapuente` here since he is already added to all directories in line 1.

```suggestion
content/events/*      @justjosie @KallieFerguson @lerongil
```